src/js/syntax-highlight.js

```diff
@@ -0,0 +1,116 @@
+import throwError from './helpers';
+
+import prism from 'prism/prism.js';
+import loadLanguages from 'prism/components/index.js';
+
+class SyntaxHighlight {
+	/**
+ * Class constructor.
+ * @param {HTMLElement|String} [messageElement] - The message element in the DOM
+ * @param {Object} [options={}] - An options object for configuring the message
```

Comment:
I think it is possible to document the properties of the options object but I do not remember how to do this